packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.test.ts

```diff
@@ -244,16 +301,22 @@ async function createUser(role: CoordinatorRole, hospitals?: Hospital[]) {
 
 function callFunction(
   userId?: string,
-  earliestTimeDays?: number
+  earliestTimeDays?: number,
+  all?: boolean
 ): Promise<FunctionsApi.GetCoordinatorAppointmentsResponse> {
   const earliestStartTimeMillis = earliestTimeDays
     ? new Date().getTime() - earliestTimeDays * 24 * 60 * 60 * 1000
     : undefined;
 
-  const request: FunctionsApi.GetCoordinatorAppointmentsRequest = {
+  let request: FunctionsApi.GetCoordinatorAppointmentsRequest = {
     hospital: Hospital.TEL_HASHOMER,
     earliestStartTimeMillis,
   };
+
+  if (all) {
```

Comment:
This is a bit confusing. Instead of having a default hospital and then overriding it from the arguments, I suggest you just ask for the hospital explicitly.

Even if this means you have to go over all existing calls and add TEL_HASHOMER to them

Reply:
done